### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by adenek** [Comfy-Org/ComfyUI#2296] [General]
Just a message to thanks the team that working on ComfyUI. The tool is now part of my daily workflow and I love it. Keep up the amazing work.

**Comment by RudyB24** [comment]
I like to second that: a big **THANK YOU** to you (and your team ?), for having created this absolutely wonderful tool, which I guess in the mean time thousands of people all over the world are using with great pleasure (and sometimes frustration :).

**Comment by othoap** [comment]
Me too, thank you all for the hard work, skill, and dedication to ComfyUI.
